inspirehep/modules/records/jsonschemas/records/elements/reference.json

```diff
@@ -0,0 +1,163 @@
+{
+    "description": "An article or thesis or book or...",
```

Comment:
Shall we change the description to say this is the schema for a reference in literature?